frontend/src/lib/k8s/namespace.ts

```diff
@@ -13,6 +13,18 @@ class Namespace extends makeKubeObject<KubeNamespace>('namespace') {
   get status() {
     return this.jsonData!.status;
   }
+
+  static isValidNamespaceFormat(namespace: string) {
+    // We allow empty strings just because that's the default value in our case.
+    if (!namespace) {
+      return true;
+    }
+
+    // Validates that the namespace is a valid DNS-1123 label and returns a boolean.
+    // https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names
+    const regex = new RegExp('^[a-z0-9]([-a-z0-9]*[a-z0-9])?$');
```

Comment:
There is this rule which the function does not validate:
"contain at most 63 characters"

Reply:
fixed!